app/classifier/classifier.jsx

```diff
@@ -253,15 +253,17 @@ class Classifier extends React.Component {
     if (!isCmdClick) {
       e.preventDefault();
     }
-    actions.classify.updateClassification({
+    actions.classify.updateMetadata({
       viewport: {
         width: innerWidth,
         height: innerHeight
       },
       interventions: {
         message: !!showIntervention,
         opt_in: !!user && user.intervention_notifications
-      }
+      },
+      workflow_translation_id: translations.strings.workflow.id,
+      user_language: translations.locale
```

Comment:
Small point, but I think I forgot to add tests for this change.

Reply:
Just added.